src/main/java/module-info.java

```diff
@@ -0,0 +1,11 @@
+module jackson_core {
```

Comment:
Presumably no way to indicate "just include all"? (perhaps with potential blacklisting of things NOT to export)